packages/netlify-cms-core/src/actions/editorialWorkflow.ts

```diff
@@ -192,69 +180,55 @@ function unpublishedEntryStatusChangePersisted(
       oldStatus,
```

Comment:
Can we remove `oldStatus`? Doesn't seems like we're using it:
https://github.com/netlify/netlify-cms/pull/4289/files#diff-c04b7dd92be649549b8b8806542af69cR121